lib/qualtrics_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module QualtricsAPI
-  VERSION = "0.0.16".freeze
+  VERSION = "0.1.0".freeze
```

Comment:
In this PR we did two things: 1. fix Kernel for Ruby 3 and 2. update bundler.
For the bundler update, we consider modern tooling compatibility to be a feature, hence we bumped a minor version instead of a patch version.

[The semantic guideline we followed](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api)